### DISCUSSION TITLE
Days Numbering

### DISCUSSION THREAD

**Discussion by EddieSpaghettie** [BJReplay/ha-solcast-solar#70] [General]
Days numbering needs to be adjusted I think. Days 3/4/5/6/7 should be 2/3/4/5/6

Today then Tomorrow then day 2. Not day 3.

Thoughts?

**Comment by autoSteve** [comment]
I think we're stuck with it, with how the sensors are named. So it is what it is.

Day 1 = today, 2 = tomorrow, etc.

**Reply by gcoan** [reply]
Agreed, it is what it is. Would be a breaking change for not a lot of value.

Might be worth just explaining it in the readme
